### ISSUE TITLE
Problem with minipages and manuscript apparatus

### ISSUE THREAD

**Issue by baem2** [maieul/ledmac#595]
When using two minipages, in the second one the manuscript apparatus will be incorrect. MWE:

```
\documentclass{article}
\usepackage{polyglossia,fontspec,xunicode}
\usepackage{libertineotf}
\setmainlanguage{latin}
\setotherlanguage{english}

\usepackage[noend,noeledsec,nofamiliar]{reledmac}

\begin{document}

\begin{minipage}[t]{0.95\linewidth}%
\beginnumbering
\pstart
\msdata{G T U V} Lorem dolor sit amet, consectetur adipiscing elit. 
 Fusce sed dolor libero. Aenean rutrum vestibulum lacus ut pretium. Fusce et auctor lectus. Ut et commodo quam, quis gravida orci. Nullam at risus elementum, suscipit enim a, pellentesque mi. 
\msdata{U V} Morbi commodo, ligula vel consectetur accumsan, massa metus egestas velit, eu fringilla leo ante in turpis. Vivamus ut tellus sollicitudin, facilisis ipsum sit amet, tincidunt odio. Maecenas tincidunt dolor sed ante blandit tincidunt. Etiam vulputate ultricies facilisis.
\pend
\endnumbering
\end{minipage}%


Other text.

\begin{minipage}[t]{0.95\linewidth}%
\beginnumbering
\pstart
\msdata{G T U V} Lorem dolor sit amet, consectetur adipiscing elit. 
 Fusce sed dolor libero. Aenean rutrum vestibulum lacus ut pretium. Fusce et auctor lectus. Ut et commodo quam, quis gravida orci. Nullam at risus elementum, suscipit enim a, pellentesque mi. 
\msdata{U V} Morbi commodo, ligula vel consectetur accumsan, massa metus egestas velit, eu fringilla leo ante in turpis. Vivamus ut tellus sollicitudin, facilisis ipsum sit amet, tincidunt odio. Maecenas tincidunt dolor sed ante blandit tincidunt. Etiam vulputate ultricies facilisis.
\pend
\endnumbering
\end{minipage}%

\end{document}
```

**Comment by maieul** [comment]
I will look on it. But be careful: you should not add space after \msdata ;-)

**Comment by maieul** [comment]
the problem was in reality with multiple \beginnumbering. Please try branch issue595.

**Comment by baem2** [comment]
Fix confirmed, thanks.

(And thanks for that hint with space as well!)

**Comment by maieul** [comment]
your welcome. will be published on CTAN as v2.16.7-2.15.0, I hope next week.
